### DISCUSSION TITLE
How should we manage global states?

### DISCUSSION THREAD

**Discussion by NadieFiind** [jabezborja/PyNani#34] [Q&A]
I've been working on PyFyre's state management. I think it's pretty good so far. Simple and effective. However, there is still one more thing that we have to deal with, which is global states.

I created my personal website using PyFyre. I defined some global states in there just by setting them as global variables. I know this is not scalable so I am thinking of a way to effectively store global states. Should we implement like a store or something?

I don't really know much about UI state management. Any suggestion is appreciated. Thanks!

**Comment by jabezborja** [comment]
In my opinion, definitely. We could use Sveltejs approach to state management

https://auth0.com/blog/state-management-in-svelte-applications/
